templating/namespaced_paths.rst

```diff
@@ -14,7 +14,7 @@ Take the following paths as an example:
 
 .. code-block:: twig
 
-    {% extends "AppBundle::layout.html.twig" %}
+    {% extends "AppBundle:layout.html.twig" %}
```

Comment:
Yeah I think this is the point. Having a layout at the `src/AppBundle/Resources/views/layout.html` to be extended by all actions like the `AppBundle:Foo:bar.html.twig` below. Looks good to me, it could extend a common `::base.html.twig`.

But I'd say that now we dropped support for 2.3, we could directly use twig notation: `@App/layout.html.twig` and `@App/Foo/bar.html.twig`, wdyt?